eisbuk-admin/cypress/plugins/index.ts

```diff
@@ -21,14 +18,17 @@ const {
  * @type {Cypress.PluginConfig}
  */
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
-module.exports = (on) => {
-  on("before:run", async (details) => {
-    console.log("override before:run");
-    await beforeRunHook(details);
-  });
+module.exports = (on: Cypress.PluginEvents) => {
```

Comment:
Did the mochawesome reporter interfere with this? Can it be re-enabled?

Reply:
Not at all...it just produced some errors when using `cypress run` (since it was unreachable/not defined in that environment) so I commented it out. Aand later on, I guess I deleted it by mistake (since it was commented out). I'll go ahead and reinstantiate that.